examples/analog_out/cont_gen_voltage_wfm_int_clk.py

```diff
@@ -0,0 +1,72 @@
+"""Example of analog output voltage generation.
+
+This example demonstrates how to output a continuous periodic
+waveform using an internal sample clock.
+"""
+
+from typing import Tuple
+
+import numpy as np
+import numpy.typing
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType
+
+
+def generate_sine_wave(
+    frequency: float,
+    amplitude: float,
+    sampling_rate: float,
+    phase_in: float,
+    number_of_samples: int,
+) -> Tuple[numpy.typing.NDArray[numpy.double], float]:
```

Comment:
Since only one example cares about phase, please make phase_in optional:
- Swap number_of_samples and phase_in parameters
- Specify a default argument for phase_in
- Do not use typing.Optional, that means "can be None" and is not needed to default to 0.0
- Update all copies of this function

``` 
def generate_sine_wave(
    frequency: float,
    amplitude: float,
    sampling_rate: float,
    number_of_samples: int,
    phase_in: float = 0.0,
) -> Tuple[numpy.typing.NDArray[numpy.double], float]:
```

Reply:
`phase_in` parameter is set as optional for all examples.